### DISCUSSION TITLE
How to reset password for admin?

### DISCUSSION THREAD

**Discussion by gaganbiswas** [pocketbase/pocketbase#2861] [Q&A] [answered]
So, basically I forgot my login password for the admin. My app is currently hosted at fly.io. Is there any way to reset the password other than sending rest password link because I haven't configured that yet from the dashboard settings.

**Comment by ganigeorgiev** [comment, marked as answer]
You can start `flyctl ssh console` and then run `/pb/pocketbase admin update youremail@example.com newpassword`.

Alternatively, you can also directly change the password hash in the database as shown in https://github.com/pocketbase/pocketbase/discussions/2496#discussioncomment-5926734.

**Reply by gaganbiswas** [reply]
`Error: unknown command "admin" for "pocketbase"`

Getting this error.

**Reply by gaganbiswas** [reply]
Ok its done! I just needed to update to pocketbase v0.16.x since admin command is available from v0.16 only. Thank you 😁

**Reply by giacomocerquone** [reply]
now this is `pb superuser update ...`

**Comment by refchef** [comment]
For anyone having a similar issue even if your pocketbase is above `v.0.16.x`: Use `superuser` instead of `admin`.

**Comment by OskarPetersilie** [comment]
For anyone using [muchobien/pocketbase-docker](https://github.com/muchobien/pocketbase-docker) 

1. Connect to Container 
    - Docker only
        - `docker exec -it <container_name_or_id> /bin/sh`
    - Docker compose stack
       -   `docker compose exec <service_name> /bin/sh`
2. Reset password
   - `/usr/local/bin/pocketbase --dir /pb_data --encryptionEnv ENCRYPTION superuser update youremail@example.com newpassword`

Notes
- `--dir /pb_data` flag is required if you mount the directory like in the example of [muchobien/pocketbase-docker](https://github.com/muchobien/pocketbase-docker) 
- `--encryptionEnv` flag is required if you use encrytion config like in the example of [muchobien/pocketbase-docker](https://github.com/muchobien/pocketbase-docker) 
- `find / -name "pocketbase" -type f 2>/dev/null`  find pocketbase binary in container
-  `find / -name "pb_data" -type d 2>/dev/null` find data dirs in container
